Lib/sndhdr.py

```diff
@@ -175,23 +174,27 @@ def test_8svx(h, f):
     # Should decode it to get #channels -- assume always 1
     return '8svx', 0, 1, 0, 8
 
-tests.append(test_8svx)
 
+tests.append(test_8svx)
```

Comment:
Why this line has been moved?

Reply:
I had some weirdness with the newlines, I've fixed it.